arangodb-net-standard.Test/CollectionApi/CollectionApiClientTest.cs

```diff
@@ -1,4 +1,4 @@
-﻿using ArangoDBNetStandard;
```

Comment:
There is a change like this on the first line of each of the modified files. It's kinda weird. I can't see what's the change about. Maybe a line return thing?

Do you mind reverting those?

Reply:
Its UTF-8 byte order mark, Roslyn appears to save source files without it. Added it back to every source file.

Reply:
Now I might added it to files that didnt have it before, but at least now every file in repo has exactly the same encoding.

Reply:
Thanks 👍